doc/ldap.rst

```diff
@@ -10,31 +10,129 @@ Configure LDAP authentication
 
 This page describes the configuration of LDAP authentication for an eLabFTW instance.
 
-First go to the Sysconfig panel by click the link at the bottom left of any page or from the user menu. Then go on the LDAP tab.
+How does eLabFTW query LDAP servers?
+========================================
 
-Set the "Toggle LDAP login" to Enabled so the option to authenticate through LDAP will be available from the login page.
+LDAP stands for "Lightweight directory access protocol" and is a common protocol to query databases with user information.
+In general, queries to LDAP servers are performed via functions supplied by ``php``.
+The overall schematic for each query with the options set in the sysadmin menu looks like this
 
-Enter the domain name or IP address of the LDAP server in the "LDAP Host" field.
+1. Connect to the LDAP server (-> TLS?, host, port)
+2. Bind (login) with username and password, or anonymously (-> username, password)
+3. Search for users within a certain region of the LDAP world directory (-> base DN) and extract certain properties/fields (-> filter attribute, team name, email, firstname, lastname)
+4. Unbind (log out)
 
-LDAP Port is 389 by default. Use port 636 for LDAPS or any custom port you might have configured.
+Obviously, for this to work, eLabFTW needs information from you.
+That is what the LDAP configuration options and the next section is for.
 
-"LDAP Base DN" will probably be something like "dc=example,dc=org".
+Sysadmin LDAP settings
+======================
 
-"LDAP Username" is the admin login for LDAP server. Not necessarily admin but at least a user that can query other users.
+To find the settings, first go to the Sysconfig panel by clicking the link at the bottom left of any page or from the user menu.
```

Comment:
Link is no longer in footer.

Reply:
Wait, you don't mean this?
![image](https://user-images.githubusercontent.com/90467693/236194281-20ef39f8-d367-41bb-b22b-895c8af50e63.png)
What do you think about "To find the settings, first go to the "Sysadmin panel" by clicking the link at the bottom left of any page or from the user menu at the top right."?

Reply:
(Resolved in the main conversation)